mqtt-client-Go/on_way-tls/main.go

```diff
@@ -39,37 +40,50 @@ func main() {
 		Cafile:   *cafile,
 	}
 	client := mqttConnect(&config)
-	go sub(client, &config)
+	go subscribe(client, &config)
 	publish(client, &config)
+	time.Sleep(time.Second)
+	unSubscribe(client, &config)
+	time.Sleep(time.Second)
+	client.Disconnect(250)
 }
 
 func publish(client mqtt.Client, config *ConConfig) {
-	for {
```

Comment:
You can keep the loop statement and just add a limit on the number of iterations.

Reply:
done